code/modules/cargo/exports.dm

```diff
@@ -125,15 +130,28 @@ Credit dupes that require a lot of manual work shouldn't be removed, unless they
 		return FALSE
 	return TRUE
 
-// Called only once, when the object is actually sold by the datum.
-// Adds item's cost and amount to the current export cycle.
-// get_cost, get_amount and applies_to do not neccesary mean a successful sale.
+/**
+  * Calculates the exact export value of the object, while factoring in all the relivant variables.
+  *
+  * Called only once, when the object is actually sold by the datum.
+  * Adds item's cost and amount to the current export cycle.
+  * get_cost, get_amount and applies_to do not neccesary mean a successful sale.
+  *
+  */
 /datum/export/proc/sell_object(obj/O, datum/export_report/report, dry_run = TRUE, allowed_categories = EXPORT_CARGO , apply_elastic = TRUE)
+	///This is the value of the object, as derived from export datums.
 	var/the_cost = get_cost(O, allowed_categories , apply_elastic)
+	///This is the value of the object, as derived from material datums. Overwritten by the_cost if it's defined.
+	var/the_material_cost = get_material_cost(O)
+	///Quantity of the object in question.
 	var/amount = get_amount(O)
+	///Utilized in the pricetag component. Splits the object's profit when it has a pricetag by the specified amount.
 	var/profit_ratio = 0
-	if(amount <=0 || the_cost <=0)
+
+	if(amount <=0 || the_cost <=0 && the_material_cost <=0)
 		return FALSE
+	if(the_cost <= 1 && the_material_cost > 0)
```

Comment:
Is it intended that the cost picked here may be the lower?
Maybe something like this instead?
`if(the_material_cost > the_cost)`